utils/rpms/daos.spec

```diff
@@ -131,7 +131,11 @@ Requires: %{name}-client = %{version}-%{release}
 Requires: spdk-tools
 Requires: ndctl
 # needed to set PMem configuration goals in BIOS through control-plane
-Requires: ipmctl > 02.00.00.3816
+%if 0%{?is_opensuse}
+Requires: ipmctl >= 02.00.00.3733
+%else
+Requires: ipmctl >= 02.00.00.3820
+%endif # 0%{?is_opensuse}
```

Comment:
Why do these need to be different for each distro?  Shouldn't there just be a single lower bound that we want to be higher than?  We don't have to use versions that are specifically in each distro.  We can specify `> <version>` for any value of _version_ that we want that may or may not exist in a given distro.

Reply:
not sure I really understand your comment, opensuse most recent version is 3733, therefore the bad versions that we are trying to avoid 3809-3816 won't be released, we don't want to support older versions as they are likely not to support barlow pass.  for centos we know that the bad versions have been released so specifying 3820 or greater will avoid the bad range. setting 3820 as minimum for both will fail for suse because that version is not available on the distro, setting 3733 as minimum for both will work , but then extra ranges to preclude the bad versions will be needed and seems like an unnecessary way of doing it. I don't think we should delay in landing this PR.

Reply:
The problem is that this PR leaves SUSE systems vulnerable to the problem once they do start packaging more recent releases, doesn't it?  Can we really land a PR that doesn't achieve the specific goal?

That might mean we need to take some further action so that this PR is successful in it's goal.  Further action could mean prodding the SUSE packagers along to packaging up a newer version.  Or we have to take on packaging a newer version of ipmctl ourselves.

Reply:
the problem has been identified so the IPMCTL team won't package broken versions for suse,so , yes , this does achieve the specific goal.

Reply:
ipmctl on Leap 15 does not come from the IPMCTL team.  It comes from SUSE:

```
Information for package ipmctl:
-------------------------------
Repository     : osl152-oss                                                 
Name           : ipmctl                                                     
Version        : 02.00.00.3733-lp152.1.3                                    
Arch           : x86_64                                                     
Vendor         : openSUSE                                                   
Installed Size : 2.6 MiB                                                    
Installed      : Yes (automatically)                                        
Status         : up-to-date                                                 
Source package : ipmctl-02.00.00.3733-lp152.1.3.src                         
Summary        : Utility for managing Intel Optane persistent memory modules
Description    :                                                            
    Utility for managing Intel Optane persistent memory modules (PMem module)
    Supports functionality to:
    - Discover PMem modules on the platform.
    - Provision the platform memory configuration.
    - View and update the firmware on PMem modules.
    - Configure data-at-rest security on PMem modules.
    - Monitor PMem module health.
    - Track performance of PMem modules.
    - Debug and troubleshoot PMem modules.
```

which is at http://download.opensuse.org/distribution/leap/15.2/repo/oss/x86_64/ipmctl-02.00.00.3733-lp152.1.3.x86_64.rpm.

So there is every possibility that they, or even some other repo packages some version > 3733 that is in the range of the broken versions.  The bottom line is that the requires **MUST** be accurate to the version required and a lack of a current version supplying the necessary package needs to be addressed by other/additional means/activities.

Setting requirements based on the assumptions of what humans will or should do is fraught with horror.